Project.toml

```diff
@@ -1,6 +1,6 @@
 name = "Weave"
 uuid = "44d3d7a6-8a23-5bf8-98c5-b353f8df5ec9"
-version = "0.9.2"
+version = "0.9.3"
```

Comment:
`0.9.2` isn't released yet, so this is unnecessary this time :p

Reply:
it's released, so never mind:
https://github.com/JuliaRegistries/General/pull/10507